cloud_governance/common/mails/mail_message.py

```diff
@@ -262,17 +265,65 @@ def get_long_run_expire_alert(self, user: str, jira_id: str):
                 <p>Hi {user},</p>
                 </div>
                 <div>
-                    <p>This is a message to alert you that the cloud long run request is expired.</p>
+                    <p>This is a message to alert you that the cloud budget request is expired.</p>
                     <p>Please take an action. If you are not using the instances Terminate the instances.</p>
-                    <p>Refer to the Jira issue: <a href="https://issues.redhat.com/browse/{jira_id}" target=="_blank">{jira_id}</a></p>
                     <p>Visit the <a href="https://clouds.perfdept.aws.rhperfscale.org:5000/wiki/clouds">wiki page</a> to get more information</p>
                 </div>
-                <div style="color:gray" class="footer">
-                    <address>
-                        --<br/>
-                        Best Regards,<br/>
-                        Cloud-governance Team<br/>
-                    </address>
-                </div>
+               {self.FOOTER}
         """.strip()
         return subject, body
+
+    def get_cost_over_usage(self, **kwargs):
+        """This method returns the subject, body for cost over usage"""
+        cloud_name = kwargs.get('CloudName', 'NA')
+        over_usage_cost = kwargs.get('OverUsageCost', 'NA')
+        full_name = kwargs.get('FullName', 'NA')
+        user_cost = kwargs.get('Cost', 0)
+        subject = f'{cloud_name} Cost Over Usage alert: > {over_usage_cost} $'
+        body = f"""
+        <div>
+        Dear {full_name},
```

Comment:
If missing name put user
Also I think to change Dear to Hi

Reply:
Add to PR description 2 main parameters:
1. CRO_COST_OVER_USAGE = 500$
2. CRO_DURATION_RANGE = 30 days